packages/gafl-webapp-service/src/pages/order-complete/order-complete/__tests__/order-complete.spec.js

```diff
@@ -47,163 +61,110 @@ const mockRequest = (statusGet = () => {}, transactionGet = () => {}) => ({
-    expect(permission.currentPage).toBe(ORDER_COMPLETE.page)
+  it.each([[LICENCE_DETAILS.uri], [NEW_TRANSACTION.uri]])('addLanguageCodeToUri is called with request and %s', async uri => {
+    const request = getSampleRequest()
+    await getData(request)
+    expect(addLanguageCodeToUri).toHaveBeenCalledWith(request, uri)
   })
 
-  it('responds with the licence information page when requested', async () => {
-    await JUNIOR_LICENCE.setup()
-    salesApi.createTransaction.mockResolvedValue(JUNIOR_LICENCE.transactionResponse)
-    salesApi.finaliseTransaction.mockResolvedValue(JUNIOR_LICENCE.transactionResponse)
+  it.each(['new', 'licenceDetails'])('data outputs addLanguageCodeToUri decorated value for %s uri', async uriName => {
+    const decoratedUri = Symbol(uriName)
+    addLanguageCodeToUri.mockReturnValue(decoratedUri)
 
-    await injectWithCookies('GET', AGREED.uri)
-    await injectWithCookies('GET', ORDER_COMPLETE.uri)
-    const data = await injectWithCookies('GET', LICENCE_DETAILS.uri)
-    expect(data.statusCode).toBe(200)
-  })
+    const { uri } = await getData(getSampleRequest())
 
-  it.each([
-    [COOKIES.page, COOKIES.uri],
-    [ACCESSIBILITY_STATEMENT.page, ACCESSIBILITY_STATEMENT.uri],
-    [PRIVACY_POLICY.page, PRIVACY_POLICY.uri],
-    [REFUND_POLICY.page, REFUND_POLICY.uri]
-  ])('succesfully navigates to %s when on the order complete page', async (page, uri) => {
-    await JUNIOR_LICENCE.setup()
-    salesApi.createTransaction.mockResolvedValue(JUNIOR_LICENCE.transactionResponse)
-    salesApi.finaliseTransaction.mockResolvedValue(JUNIOR_LICENCE.transactionResponse)
-
-    await injectWithCookies('GET', AGREED.uri)
-    await injectWithCookies('GET', ORDER_COMPLETE.uri)
-
-    const data = await injectWithCookies('GET', uri)
-    expect(data.statusCode).toBe(200)
+    expect(uri[uriName]).toEqual(decoratedUri)
   })
 
-  it.each([[LICENCE_DETAILS.uri], [NEW_TRANSACTION.uri]])('addLanguageCodeToUri is called with request and %s', async uri => {
-    const status = () => ({
-      [COMPLETION_STATUS.agreed]: true,
-      [COMPLETION_STATUS.posted]: true,
-      [COMPLETION_STATUS.finalised]: true
-    })
-    const transaction = () => ({
-      startDate: '2019-12-14T00:00:00Z',
-      licensee: {
-        postalFulfilment: 'test',
-        preferredMethodOfConfirmation: 'test'
-      }
-    })
-
-    const request = mockRequest(status, transaction)
-
-    displayStartTime.mockReturnValueOnce('1:00am on 6 June 2020')
+  it.each(['http://trustpilot.com', 'http://give-us-a-stinker'])('feedback link set to FEEDBACK_URI env var (%s)', async feedbackUri => {
+    process.env.FEEDBACK_URI = feedbackUri
+    const {
+      uri: { feedback }
+    } = await getData(getSampleRequest())
+    expect(feedback).toBe(feedbackUri)
+  })
 
-    await getData(request)
-    expect(addLanguageCodeToUri).toHaveBeenCalledWith(request, uri)
+  it("uses FEEDBACK_URI_DEFAULT if FEEDBACK_URI env var isn't set", async () => {
```

Comment:
Think these should be the ' symbol

Reply:
You can't set values of process.env to be anything other than a string, so using a Symbol isn't possible, unfortunately

Reply:
Sorry I mean these are quotation marks on the each statement. However I assume this was done as you have apostrophes in the description